pkg/distro/fedora/images.go

```diff
@@ -524,6 +524,7 @@ func iotImage(workload workload.Workload,
 	img.OSName = "fedora-iot"
 
 	if !common.VersionLessThan(distro.Releasever(), "38") {
+		img.KernelOptionsAppend = append(img.KernelOptionsAppend, "coreos.no_persist_ip")
```

Comment:
Does this have no effect because the kernel option is ignored? If so I'd still like it to be conditional.